src/odemis/driver/hamamatsurx.py

```diff
@@ -349,7 +349,7 @@ def _sync(self, event):  # event = self.softwareTrigger
         """
         Synchronize the acquisition on the given event. Every time the event is
           triggered, the DataFlow will start a new acquisition.
-        Behaviour is unspecified if the acquisition is already running.  # TODO still True?
+        Behaviour is unspecified if the acquisition is already running.  # TODO still True? please comment
```

Comment:
Actually, I think this is the generic statement. That basically means that each driver can decide. In the case of this, driver, it will not work when the acquisition is already running. So you should mention that.